how-to/integrate-with-sso/integrate-with-okta/public/manifest.fin.json

```diff
@@ -6,7 +6,7 @@
 		"version": "28.106.72.17"
```

Comment:
The version number has to be updated to reflect the runtime version used by workspace 11.